### PR TITLE
Latest devel to main

### LINKED ISSUE #1: PC and PP test data needed
See https://github.com/pharmaverse/admiral/issues/827 - need to use CDISC pilot DM data to create realistic synthetic data for PC and PP in {admiral.test} to allow {admiral} development of PK ADaMs.
Once we have this we could also share back with CDISC in case they want to add to their pilot data repository.

Note that personal data shouldn't be used for testing even if anonymised. One of my colleagues co-authored this useful paper recently on the topic: https://link.springer.com/article/10.1007/s44163-021-00016-y - it also gives lots of great advice on methods around synthetic data generation.

### LINKED ISSUE #20: Add SV dataset
- Download dataset from [here](https://bitbucket-test.cdisc.org/projects/CED/repos/sdtm-adam-pilot-project/browse/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/tabulations/sdtm)
- Read into R using `haven::read_xpt()`
- Save into the `data` folder using `save(sv, file = "data/sv.rda", compress = "bzip2")`

### PR DESCRIPTION
**Pull request by rossfarrugia**
Updating main branch to add SV, PC and PP.

closes #1 & closes #20

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@thomas-neitmann this closes #2 now as well (thanks @Gopi2345!). appreciate if you could merge to main when you can, so ready for {admiralonco} dev. i updated the NEWS file and DESCRIPTION with the new contributors

**Comment by thomas-neitmann** [comment]
@rossfarrugia This may be a stupid question ehy we do we need these updates on the `main` branch?

**Review by bundfussr — CHANGES_REQUESTED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# admiraltest 2.0.0
```
[the remark this answers is not in the document]
Comment (rossfarrugia) [reply NEWS.md]: good spot @bundfussr - now updated. changed both to 1.1.0 as dont think this justifies being called a major release in hindsight

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# admiraltest 1.1.0
+
+- Extending the test data offering by adding PK datasets - PC and PP (#1) 
+- Adding Oncology test data - RS, TU, TR, SUPPTR (#2)
+- Adding SV from CDISC pilot project (#20)
+
+
+# admiraltest 1.0.0
```
Comment (bundfussr) [inline NEWS.md:8]: The version released at CRAN is 0.1.2.

Thus we should change this heading and maybe use a different version number for the new version, e.g., 0.2.0?

**Review by bundfussr — CHANGES_REQUESTED** [review]

File: NEWS.md (hunk 2 above)
Comment (rossfarrugia) [reply NEWS.md]: yes you're right, updated now and added entries to explain 0.1.1 and 0.1.2

**Review by bundfussr — APPROVED** [review]

**Pull request merged**